pkgs/applications/video/obs-studio/plugins/obs-tuna/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, obs-studio
+, cmake
+, zlib
+, curl
+, taglib
+, dbus
+, pkg-config
+, qtbase
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
I think we should replace rec with the finalAttrs pattern. See an implementation here: https://github.com/NixOS/nixpkgs/pull/237742